core/modules/user/user.module

```diff
@@ -2270,27 +2270,7 @@ function user_view_page($account) {
  *   An array as expected by backdrop_render().
  */
 function user_view($account, $view_mode = 'full', $langcode = NULL) {
```

Comment:
To keep the same pattern as in other view functions, we should user (User $account, ...) here (declare the type of argument)

Reply:
Let's make this a follow-up as well.